python_modules/dagster/dagster_tests/daemon_sensor_tests/test_sensor_failure_recovery.py

```diff
@@ -59,175 +54,148 @@ def _test_launch_sensor_runs_in_subprocess(instance_ref, execution_datetime, deb
 )
 @pytest.mark.parametrize("crash_location", ["TICK_CREATED", "TICK_HELD"])
 @pytest.mark.parametrize("crash_signal", get_crash_signals())
-def test_failure_before_run_created(crash_location, crash_signal, capfd):
+def test_failure_before_run_created(crash_location, crash_signal, instance, external_repo):
     frozen_datetime = to_timezone(
         create_pendulum_time(year=2019, month=2, day=28, hour=0, minute=0, second=1, tz="UTC"),
         "US/Central",
     )
 
-    with instance_with_sensors() as (
-        instance,
-        _grpc_server_registry,
-        external_repo,
-    ):
```

Comment:
these changes look big but are just taking this out and using a shared fixture instead